src/components/accessories/table/Table.tsx

```diff
@@ -233,11 +233,13 @@ const Table: FunctionComponent<IProps> = ({
 
   return (
     <>
-      <div className="header">
-        <Button type="button" onClick={handleExpand}>
-          {expanded ? t("common.collapse_all") : t("common.expand_all")}
-        </Button>
-      </div>
+      {isCollapsabile ?? (
+        <div className="header">
+          <Button type="button" onClick={handleExpand}>
+            {expanded ? t("common.collapse_all") : t("common.expand_all")}
+          </Button>
+        </div>
+      )}
```

Comment:
```suggestion
        <div className="header">
        {!!isCollapsabile && (
          <Button type="button" onClick={handleExpand}>
            {expanded ? t("common.collapse_all") : t("common.expand_all")}
          </Button>
          )}
        </div>
```

Reply:
Hello @gasp ! Great Job. I think with this suggestion, we could move on...

Reply:
Thanks Steve for the review and the suggestion !